src/lib/resource/sections/applicant-have-you-applied-to-us-before.js

```diff
@@ -15,14 +15,14 @@ module.exports = {
                         'applicant-have-you-applied-to-us-before': {
                             title: {
                                 myself:
-                                    'Have you applied to us before in connection with this or any other crime?',
+                                    'Have you applied to us before in connection with any other crime?',
                                 proxy: 'Has anyone applied to us before on behalf of the victim?',
                                 deceased:
                                     'Has the claimant applied to us before in connection with this or any other crime?'
```

Comment:
Should the reference to this crime still be here?

Reply:
yeah, this only affects Personal Injury claims